### PR TITLE
Merge batch lab tests

### PR DESCRIPTION
**Pull request by fredkingham**
If the upstream lab test table is using inactive MRNs, add the lab tests to the correct active MRN for that table.

While doing so remove some of the old IPL logic as this is inconsistently applied compared with other loaders.

### CODE REVIEW

**Comment by fredkingham** [comment]
I'm going to move this back into in progress as I think we should use `get_patient` from https://github.com/openhealthcare/elcid-rfh/pull/1361

File: intrahospital_api/loader.py [hunk 1]
```diff
@@ -296,44 +296,38 @@ def _batch_load():
 @transaction.atomic
 def update_patient_from_batch(demographics_set, data_delta):
```
Comment (davidmiller) [inline intrahospital_api/loader.py:297]: Are you sure this is the function that actually runs in production?

Comment (fredkingham) [reply intrahospital_api/loader.py:297]: good call, closing this for https://github.com/openhealthcare/elcid-rfh/pull/1390

**Comment by fredkingham** [comment]
closing for https://github.com/openhealthcare/elcid-rfh/pull/1390

**Pull request closed without merging**